index.php

```diff
@@ -144,7 +144,7 @@ function send_response($input_raw) {
                       "date": 1440704423,
                       "text": "@CMNisal, Which farm do you want the details of?"
                     },
-                    "text": "Cancel"
+                    "text": "/activefarms"
```

Comment:
Wouldn't this allow anyone to view all active farms? Can we limit this to Bot Dev group only?